src/module.c

```diff
@@ -502,6 +502,11 @@ char *RM_Strdup(const char *str) {
     return zstrdup(str);
 }
 
+/* Use like zmalloc_usable_size(), returns usable size of given ptr. */
+size_t RM_AllocUsableSize(void *ptr) {
+    return zmalloc_usable_size(ptr);
+}
```

Comment:
let's move it next to RM_MallocSize, and maybe call it RM_MallocUsableSize or RM_MallocSizeUsable and explain the difference.
i.e. one is the consumed amount of memory, and the other is what can be used by the module.

Reply:
done